support/cas-server-support-spnego-webflow/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
     implementation project(":core:cas-server-core-services-api")
     implementation project(":support:cas-server-support-spnego")
     implementation project(":support:cas-server-support-ldap-core")
+    implementation project(":support:cas-server-support-pm-webflow")
```

Comment:
You should try to switch this to `compileOnly`. Leaking the PM module statically for those who run SPNEGO who might have no use for PM is not a good idea. The reference should only be done for compile-time, by a class-name reference or you actually can order webflow configurers IIRC so one executes before the other without changes to the source code.

Reply:
In fact, the last option is probably best out of all since it will be non-maintainable to include module references in each project and cross-reference them as such to control the order. (i.e. imagine if some other module had to execute after spnego, before PM but after X and before Y. This will quickly get out of hand). 

A `CasWebflowConfigurer` can be ordered using the `Ordered` interface; We should just ensure the order is exposed somehow using a setting so that you can control the execution of the webflow configurer dynamically without changing the source code.

Reply:
I think `compileOnly` is not enough as it will be checked at runtime.

In fact, this way to do only works for the PM support and another support as the PM support is by default in the CAS server.
But you're right, it does not scale for other modules if one of them is not the PM.

I will switch to the `Ordered` interface, it seems to be a much better solution.

Reply:
I updated the PR accordingly...